### PR TITLE
Draft event page - july comm call

### PR DESCRIPTION
**Pull request by yabellini**

### CODE REVIEW

File: content/commcalls/2023-07-31/index.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+resources:
+  - url: 
+    name:   
+    details: 
+
+speakers:  
+  - name: Beatriz Milz
+    bio: 
+    img: /img/community/picture.jpeg
+  - name:
```
Comment (yabellini) [inline content/commcalls/2023-07-31/index.md:32]:
```suggestion
  - name: Victor Ordu
```

File: content/commcalls/2023-07-31/index.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    details: 
+
+speakers:  
+  - name: Beatriz Milz
+    bio: 
+    img: /img/community/picture.jpeg
+  - name: Victor Ordu
+    bio:        
+    img: /img/community/picture.jpg    
+  - name:
```
Comment (yabellini) [inline content/commcalls/2023-07-31/index.md:35]:
```suggestion
  - name:  Carolina Pradier
```

File: content/commcalls/2023-07-31/index.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+attendees: All are welcome
+slug: july2023-championprogram
+notes: 
+outputs:
+- HTML
+- Calendar 
+# vimeo:
+# - id: 
+resources:
+  - url:
```
Comment (yabellini) [inline content/commcalls/2023-07-31/index.md:24]:
```suggestion
  - url: https://github.com/BroVic/naijR
```

File: content/commcalls/2023-07-31/index.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+slug: july2023-championprogram
+notes: 
+outputs:
+- HTML
+- Calendar 
+# vimeo:
+# - id: 
+resources:
+  - url: 
+    name:
```
Comment (yabellini) [inline content/commcalls/2023-07-31/index.md:25]:
```suggestion
    name:   R package on Nigeria and for Nigeria 
```

File: content/commcalls/2023-07-31/index.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+notes: 
+outputs:
+- HTML
+- Calendar 
+# vimeo:
+# - id: 
+resources:
+  - url: 
+    name:   
+    details:
```
Comment (yabellini) [inline content/commcalls/2023-07-31/index.md:26]:
```suggestion
    details: The goal of naijR is to make it easier for R users to work with data related to Nigeria. This package is going through the rOpenSci software peer review. 
```

**Review by yabellini — COMMENTED** [review]
Adding Victor package info

File: content/commcalls/2023-07-31/index.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+outputs:
+- HTML
+- Calendar 
+# vimeo:
+# - id: 
+resources:
+  - url: https://github.com/BroVic/naijR
+    name:   R package on Nigeria and for Nigeria 
+    details: The goal of naijR is to make it easier for R users to work with data related to Nigeria. This package is going through the rOpenSci software peer review. 
+
```
Comment (yabellini) [inline content/commcalls/2023-07-31/index.md:27]:
```suggestion
resources:
  - url: https://github.com/holatam/eph
    name:  eph - Toolbox for the processing of the Permanent Household Survey in Argentina
    details: package to download and manipulate the Permanent Household Survey from Argentina (eph is the Spanish acronym for Permanent Household Survey)
```

**Comment by yabellini** [comment]
Please, @caropradier, @beatrizmilz and @BroVic can you review if the events look fine? I summarized a bio for each one of you, and added a link to the packages of Caro and Victor on resources. 

Here you can view the preview: https://deploy-preview-586--ropensci.netlify.app/

I will publish this after we choose date and time. Thanks in advance!

File: content/commcalls/2023-07-31/index.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+outputs:
+- HTML
+- Calendar 
+# vimeo:
+# - id: 
+resources:
+  - url: https://github.com/BroVic/naijR
+    name:   R package on Nigeria and for Nigeria 
+    details: The goal of naijR is to make it easier for R users to work with data related to Nigeria. This package is going through the rOpenSci software peer review. 
+resources:
```
Comment (yabellini) [inline content/commcalls/2023-07-31/index.md:27]:
```suggestion

```

**Comment by caropradier** [comment]
Looks good Yani! 
What about the date? I thought we were organizing the event the previous week

**Comment by yabellini** [comment]
> What about the date? I thought we were organizing the event the previous week

Yes Caro! I always use the last date of the month until we agree on a date and time and then I change it before publishing.

**Comment by BroVic** [comment]
Hello @yabellini, it looks good but I have a few suggestions for the para referring to me. You could replace "_I also conducts R training_..." with "_I also **conduct** R training_...". Then you could replace the comma after "_author of R packages_" with a semicolon.

File: content/commcalls/2023-07-31/index.md [hunk 8]
```diff
@@ -0,0 +1,58 @@
+---
+title: "Mentoring & training program for Scientific Open Source Champions"
+dateStart: 2023-07-31T20:00:00 # UTC!!
```
Comment (yabellini) [inline content/commcalls/2023-07-31/index.md:3]:
```suggestion
dateStart: 2023-07-25T14:00:00 # UTC!!
```

File: content/commcalls/2023-07-31/index.md [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+dateEnd: 2023-07-31T21:00:00 # UTC!!
```
Comment (yabellini) [inline content/commcalls/2023-07-31/index.md:4]:
```suggestion
dateEnd: 2023-07-25T15:00:00 # UTC!!
```

File: content/commcalls/2023-07-31/index.md [hunk 10]
```diff
@@ -0,0 +1,58 @@
...
+date: 2023-07-31T21:00:00 # UTC!! same as dateEnd
```
Comment (yabellini) [inline content/commcalls/2023-07-31/index.md:5]:
```suggestion
date: 2023-07-31T15:00:00 # UTC!! same as dateEnd
```

File: content/commcalls/2023-07-31/index.md [hunk 11]
```diff
@@ -0,0 +1,58 @@
...
+  - url: https://github.com/holatam/eph
+    name:  eph - Toolbox for the processing of the Permanent Household Survey in Argentina
+    details: package to download and manipulate the Permanent Household Survey from Argentina (eph is the Spanish acronym for Permanent Household Survey)
+speakers:  
+  - name: Beatriz Milz
+    bio: I'm a mentor in the rOpenSci's Champions Program . I'm an R-Lady from São Paulo - Brazil and a Ph.D. Candidate in Environmental Sciences at University of São Paulo and I also work as a Data Scientist and an R teacher at Curso-R. I am a tidyverse instructor certified by Posit, and a co-organizer of communities and events, such as Latin-R, and satRday São Paulo.
+    img: /img/team/beatriz-milz-mentor.jpeg
+  - name: Victor Ordu
+    bio: I'm a medical doctor with many years of experience in the Nigerian healthcare space. I'm an independent consultant for the UN and international NGOs working in Nigeria, R has been the focus of much of my work. I also conducts R training programs for public and private organizations in Nigeria and I'm the author of R packages, one of them is part of my project as rOpenSci Champion.
```
Comment (yabellini) [inline content/commcalls/2023-07-31/index.md:35]:
```suggestion
    bio: I'm a medical doctor with many years of experience in the Nigerian healthcare space. I'm an independent consultant for the UN and international NGOs working in Nigeria, and R has been the focus of much of my work. I also conduct R training programs for public and private organizations in Nigeria. I'm the author of R packages; one is part of my project as rOpenSci Champion.        
```

**Review by yabellini — COMMENTED** [review]
Applying Victor suggestion and updating date

File: content/commcalls/2023-07-31/index.md [hunk 12]
```diff
@@ -0,0 +1,58 @@
...
+    bio:  I am an economist and data scientist. I have a degree in Economics, and I am currently finishing my Master’s degree in Labour Studies, both at the University of Buenos Aires. I work as a research and teaching assistant at CEPED-UBA and I’m also an active member of Ecofeminita’s economics and data team (ecofemidata) and maintainer of the eph R package, which is my project as rOpenSci Champion. 
+    img: /img/team/carolina-pradier-mentee.jpg  
+tags:
+  - community call
+  - events
+  - champions program
+deets: |
+    Meeting ID:  
+    Passcode: 
+zoomurl:
```
Comment (yabellini) [inline content/commcalls/2023-07-31/index.md:47]:
```suggestion
zoomurl: https://numfocus-org.zoom.us/j/87549477466?pwd=b0cydVpIYW5yMm9MKzZONG16RTVsUT09
```

File: content/commcalls/2023-07-31/index.md [hunk 13]
```diff
@@ -0,0 +1,58 @@
...
+    img: /img/team/victor-ordu-mentee.jpg  
+  - name:  Carolina Pradier
+    bio:  I am an economist and data scientist. I have a degree in Economics, and I am currently finishing my Master’s degree in Labour Studies, both at the University of Buenos Aires. I work as a research and teaching assistant at CEPED-UBA and I’m also an active member of Ecofeminita’s economics and data team (ecofemidata) and maintainer of the eph R package, which is my project as rOpenSci Champion. 
+    img: /img/team/carolina-pradier-mentee.jpg  
+tags:
+  - community call
+  - events
+  - champions program
+deets: |
+    Meeting ID:
```
Comment (yabellini) [inline content/commcalls/2023-07-31/index.md:45]:
```suggestion
    Meeting ID:  875-494-77466
```

File: content/commcalls/2023-07-31/index.md [hunk 14]
```diff
@@ -0,0 +1,58 @@
...
+  - name:  Carolina Pradier
+    bio:  I am an economist and data scientist. I have a degree in Economics, and I am currently finishing my Master’s degree in Labour Studies, both at the University of Buenos Aires. I work as a research and teaching assistant at CEPED-UBA and I’m also an active member of Ecofeminita’s economics and data team (ecofemidata) and maintainer of the eph R package, which is my project as rOpenSci Champion. 
+    img: /img/team/carolina-pradier-mentee.jpg  
+tags:
+  - community call
+  - events
+  - champions program
+deets: |
+    Meeting ID:  
+    Passcode:
```
Comment (yabellini) [inline content/commcalls/2023-07-31/index.md:46]:
```suggestion
    Passcode: 516675
```

File: content/commcalls/2023-07-31/index.md [hunk 15]
```diff
@@ -0,0 +1,58 @@
...
+resources:
+  - url: https://github.com/BroVic/naijR
+    name:   R package on Nigeria and for Nigeria 
+    details: The goal of naijR is to make it easier for R users to work with data related to Nigeria. This package is going through the rOpenSci software peer review. 
+  - url: https://github.com/holatam/eph
+    name:  eph - Toolbox for the processing of the Permanent Household Survey in Argentina
+    details: package to download and manipulate the Permanent Household Survey from Argentina (eph is the Spanish acronym for Permanent Household Survey)
+speakers:  
+  - name: Beatriz Milz
+    bio: I'm a mentor in the rOpenSci's Champions Program . I'm an R-Lady from São Paulo - Brazil and a Ph.D. Candidate in Environmental Sciences at University of São Paulo and I also work as a Data Scientist and an R teacher at Curso-R. I am a tidyverse instructor certified by Posit, and a co-organizer of communities and events, such as Latin-R, and satRday São Paulo.
```
Comment (beatrizmilz) [inline content/commcalls/2023-07-31/index.md:32]: I think would be good to cite that I'm a R-Ladies Co-organizer.
```suggestion
    bio: I'm a mentor in the rOpenSci's Champions Program. I'm an R-Ladies co-organizer in São Paulo - Brazil and a Ph.D. Candidate in Environmental Sciences at University of São Paulo. I also work as a Data Scientist and an R teacher at Curso-R. I am a tidyverse instructor certified by Posit, and a co-organizer of communities and events, such as Latin-R, R-Ladies São Paulo and satRday São Paulo.
```

**Review by beatrizmilz — APPROVED** [review]

File: content/commcalls/2023-07-31/index.md [hunk 16]
```diff
@@ -0,0 +1,58 @@
...
+# vimeo:
+# - id: 
+resources:
+  - url: https://github.com/BroVic/naijR
+    name:   R package on Nigeria and for Nigeria 
+    details: The goal of naijR is to make it easier for R users to work with data related to Nigeria. This package is going through the rOpenSci software peer review. 
+  - url: https://github.com/holatam/eph
+    name:  eph - Toolbox for the processing of the Permanent Household Survey in Argentina
+    details: package to download and manipulate the Permanent Household Survey from Argentina (eph is the Spanish acronym for Permanent Household Survey)
+speakers:
```
Comment (yabellini) [inline content/commcalls/2023-07-31/index.md:30]:
```suggestion
  - url: /champions/
    name:   Champions Program
    details: web page with details about the champion program.
speakers:  
```

**Pull request merged**